docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md

```diff
@@ -0,0 +1,67 @@
+# Week 5 Todos
+
+## The story so far
+
+By now your entire team should have done a team retrospective. Meetings are hard
+so please try to think of ways to collaborate async with your team members. For
+example, use Miro to collect ideas async and shorten meetings with clear agenda.
+
+Please continue to promote your members efforts using the 'Shoutout!' channel or
+team meetings. It helps to drive engagement and improve team morale.
+
+## Planing
+
+Please organise time for presentation and preparing the materials for the
+Showcase this week.
+
+Remind team to start looking at the Software Requirements Specification in
+preparation for OnTrack Task 6.2.
+
+Continue to build out your Trello boards. For teams who are still struggling
+with this, we have prepared a Trello
+[guide](https://github.com/thoth-tech/handbook/blob/main/docs/learning/training/trello-guide.md).
+We will run a short workshop later this week to cover in more detail.
+
+If you are would like to improve sprint planning, the upcoming delivery lead
+workshop might be of great interest. We have rescheduled it from 6 - 7PM on
+Tuesday 05/04/2022.
+
+Guidance on user stories and requirement extracting will also be coming later
+this week.
+
+## Calendar
+
+Please use our company shared calendar
+[\_#Thoth-Tech](https://outlook.office.com/calendar/group/deakin365.onmicrosoft.com/_thoth-tech/view/month)
+for regular team meetings, workshops, company meeting/event info. If you have
+any difficulties with this, let us know.
+
+## Leadership support (you):
+
+You could help the company to help us understand the company learning and
+development needs and plan for resources. Please advocate the
+[survey](https://forms.office.com/Pages/ResponsePage.aspx?id=7Hgj0IgW1UaFQBwotfRw9pxmFQ5mP6pJsCNkD-HdouBUNUVBVlBEMlVTT0RKWkdVNTJOT0o5MTNMTSQlQCN0PWcu)
+to your team. If any team members would like to facilitate any training or you
+have identified any experts that can spread knowledge beyond your teams, please
+do not hesitate to reach out to the Area Leads. We love to collaborate and build
+a great learning culture at Thoth-tech.
+
```

Comment:
If you would like to express interest to attend the leadership conference - please let us know. This is a shadowing development experience that would be beneficial, particularly for juniors taking more leadership responsibility next trimester

Reply:
Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)